apps/nestjs-backend/src/features/record/record.service.ts

```diff
@@ -335,7 +334,6 @@ export class RecordService implements IAdapterService {
     const view = await this.getTinyView(tableId, viewId);
 
     const filter = mergeWithDefaultFilter(view?.filter, extraFilter);
-    const orderBy = mergeWithDefaultSort(view?.sort, extraOrderBy);
```

Comment:
Instead of ignoring `view.sort`, maybe you should put `extraOrderB`y priority first.